allennlp/nn/util.py

```diff
@@ -1786,8 +1786,8 @@ def masked_topk(
     k = k.reshape(-1)
```

Comment:
There's a `1e-13` above that was chosen specifically for regular precision.  It should be addressed somehow for amp.  Maybe with an `EPSILON` value.